spec/draft/profile-spec.md

```diff
@@ -23,7 +23,7 @@ When submitted to a profile store, the profile must be assigned a globally uniqu
 
 # Profile Document
 
-ProfileDocument : Description? ProfileName ProfileVersion Usecase+ NamedModel* NamedField*
+ProfileDocument : Description? ProfileName ProfileVersion Usecase+ NamedModel* NamedField* Example*
```

Comment:
@smizell shouldn't the example be on the Usecase level? How do you attach it to a particular use case the case of multiple usecases per profile?